app/ts/background/background-startup.ts

```diff
@@ -182,7 +182,7 @@ async function startup() {
 	const settings = await getSettings()
 	const userSpecifiedSimulatorNetwork = settings.currentRpcNetwork.httpsRpc === undefined ? await getPrimaryRpcForChain(1n) : settings.currentRpcNetwork
 	const simulatorNetwork = userSpecifiedSimulatorNetwork === undefined ? defaultRpcs[0] : userSpecifiedSimulatorNetwork
-	const simulator = new Simulator(simulatorNetwork, newBlockAttemptCallback, onErrorBlockCallback)
+	const simulator = new Simulator(simulatorNetwork, newBlockAttemptCallback, onErrorBlockCallback, 60000)
```

Comment:
`tokenPriceCacheAge` defaults to 60000 already so can be omitted

```suggestion
	const simulator = new Simulator(simulatorNetwork, newBlockAttemptCallback, onErrorBlockCallback)
```